kiwi/builder/container.py

```diff
@@ -69,6 +71,14 @@ def __init__(self, xml_state, target_dir, root_dir):
                 '.', self.requested_container_type, '.tar.xz'
             ]
         )
+        self.base_image = None
+
+        base_image = xml_state.build_type.get_derived_from()
+        if base_image:
+            uri = Uri(base_image, 'images')
+            self.base_image = os.sep.join(
+                [self.root_dir, 'image', os.path.basename(uri.translate())]
+            )
```

Comment:
Can we make the variable naming more clear ? imho base_image is used to hold just the value of the derived_from attribute and after that its meaning changes to a local path from an uri translate

Second an uri describes a resource which must not necessarily be a local path. Thus I think an uri.is_remote() check seems to be needed here too

Thoughts ?